bundles/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/action/VeluxActions.java

```diff
@@ -91,12 +91,8 @@ public void setThingHandler(@Nullable ThingHandler handler) {
      * @throws IllegalArgumentException if actions is invalid
      * @throws IllegalStateException if anything else is wrong
      */
-    public static Boolean rebootBridge(@Nullable ThingActions actions)
-            throws IllegalArgumentException, IllegalStateException {
-        if (!(actions instanceof IVeluxActions)) {
-            throw new IllegalArgumentException("Unsupported action");
-        }
-        return ((IVeluxActions) actions).rebootBridge();
+    public static Boolean rebootBridge(ThingActions actions) throws IllegalArgumentException, IllegalStateException {
+        return ((VeluxActions) actions).rebootBridge();
```

Comment:
Please undo, the check of actions is necessary for DSL rules (in particular to avoid a NPE).

Reply:
Ok. Done.